.github/workflows/f5-cla.yml

```diff
@@ -0,0 +1,52 @@
+name: F5 CLA
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_target:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+concurrency:
```

Comment:
Good call on aborting concurrent workflows, I'll push this up to the sample workflow file.